packages/api/controllers/project.js

```diff
@@ -12,10 +12,9 @@ export async function createProject({
   if (!project) throw new Error('project is required')
 
   const id = generateId()
-  const partitionKey = getPk({ userId, programId })
   const item = {
     ...project,
-    programKey: partitionKey,
+    programId,
     id,
```

Comment:
So.... If it's not too much trouble.... Could we change all of these `id`s to (for example) `projectId`? I should have done this from the start.

Reply:
I thought we made `programId` as a partition key for `Project` table just to make DynamoDb store the program's projects close to each other, which will make the fetch more efficient.

Reply:
Yeah I'm just talking about the `id` field, rename it to the more explicit `projectId`